packages/application/test/shell.spec.ts

```diff
@@ -8,6 +8,7 @@ import { CommandRegistry } from '@lumino/commands';
 import { Message } from '@lumino/messaging';
 import { DockPanel, Widget } from '@lumino/widgets';
 import { simulate } from 'simulate-event';
+import { ACTIVE_CLASS, CURRENT_CLASS } from '../src/shell';
```

Comment:
I think that the pattern elsewhere is not to export class names but re-define them in the tests, e.g.:

https://github.com/jupyterlab/jupyterlab/blob/63f32800e989c0113d1b8caec34d4f530e094e74/packages/metapackage/test/completer/widget.spec.ts#L25

It reduces the public API and works as a good sanity check.